apps/intellij/src/main/kotlin/dev/nx/console/nxls/WorkspacePathSetting.kt

```diff
@@ -0,0 +1,33 @@
+package dev.nx.console.nxls
+
+import com.intellij.ui.components.JBTextField
+import com.intellij.ui.dsl.builder.MAX_LINE_LENGTH_WORD_WRAP
+import com.intellij.ui.dsl.builder.Panel
+import com.intellij.ui.dsl.gridLayout.HorizontalAlign
+import dev.nx.console.settings.NxConsoleSettingBase
+
+class WorkspacePathSetting : NxConsoleSettingBase<String?> {
+
+    private val inputField = JBTextField()
```

Comment:
Maybe change to a Field with Browse Button

Reply:
Yes this is planned (why the PR is only draft for now)